### PR TITLE
welcome text center

### PR DESCRIPTION
**Pull request by AbhijithNT**
# Pull Request

### Description
Screenshot from 2021-09-19 11-19-21

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
Thank you for your contribution! LGTM. 🚀

File: public/index.css [hunk 1]
```diff
@@ -1,144 +1,144 @@
...
-	height: 97.5vh;
-	text-align: left;
-	align-items: center;
-	font-size: 27px;
-	background: #3a3240;
+  display: flex;
+  height: 97.5vh;
+  font-size: 27px;
+  justify-content: center;
+  align-items: center;
```
Comment (Akshu-on-github) [inline public/index.css:74]: The changes LGTM - Just need to check with @nhcarrigan if the tab space is okay?

Comment (nhcarrigan) [reply public/index.css:74]: I should set up some configs on this.

**Review by nhcarrigan — APPROVED** [review]
Looks good to me.

**Comment by nhcarrigan** [comment]
## Congratulations

Congratulations on making your first pull request to our project. We have reviewed your contributions and are happy to accept them.

We hope you enjoyed the contribution process, and encourage you to watch for issues labelled `help wanted` for additional opportunities to contribute.

**Pull request merged**